gitspy/Parseable.swift

```diff
@@ -9,7 +9,31 @@
 import Foundation
 
 protocol Parseable {
-    typealias A
-    func parse(data: AnyObject) -> A?
+    
+    init?(dictionary: [String: AnyObject])
+    init?(data: AnyObject)
+    
+    // there could be a cooler way with Array extension + generics...
+    static func array(objects: [[String: AnyObject]]) -> [Self]
 }
 
+extension Parseable {
+    
+    init?(data: AnyObject) {
+        guard let dictionary = data as? [String: AnyObject] else {
+            return nil
+        }
+        self.init(dictionary: dictionary)
+    }
+
+    static func array(objects: [[String: AnyObject]]) -> [Self] {
+        var array = [Self]()
+        for dictionary in objects {
+            if let object = Self(dictionary: dictionary) {
+                array.append(object)
+            }
+        }
+        return array
+    }
```

Comment:
Don't like it either (not the static, note: static here is the equivalent of class method for classes, is not the same as static in C (see discussion on swift-evolution to rename it to `type` https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20151207/000811.html)).
As I mentioned in a comment somewhere, it should be possible somehow to create an extension of `Array`, with generics, that create an array of `Parseable` objects, and would be more clean, I guess. But I wasn't able to do it, I will retry if you think it can be okay